app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -90,35 +90,7 @@ public boolean onPrepareActionMode(ActionMode inputMode, Menu inputMenu) {
 
         @Override
         public boolean onActionItemClicked(ActionMode inputMode, MenuItem inputItem) {
-            if (inputItem.getItemId() == R.id.action_copy_to_clipboard) {
-                ClipboardManager clipboard = (ClipboardManager) getSystemService(CLIPBOARD_SERVICE);
```

Comment:
This leaves a few unused imports like `android.content.ClipData` and `android.content.ClipboardManager` (haven't checked for more).